bin/check-diff

```diff
@@ -1,11 +0,0 @@
-#!/bin/bash
-
-# Make sure we notice any untracked files generated by the build
-git add --intent-to-add .
-git diff --quiet dist/
-retVal=$?
-if [ $retVal -ne 0 ]; then
-  echo "Detected uncommitted changes after build:"
-  git --no-pager diff dist/
```

Comment:
The contents of the `diff/` folder are marked as [generated](https://github.com/dependabot/fetch-metadata/blob/main/.gitattributes) so this was just to ensure we spat out the actual change in the obfuscated JS.